ffi/examples/visit-expression/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.12)
+project(visit_expressions)
+
+add_executable(visit_expression visit_expression.c)
+target_compile_definitions(visit_expression PUBLIC DEFINE_DEFAULT_ENGINE)
+target_include_directories(visit_expression PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../../../target/ffi-headers")
+target_link_directories(visit_expression PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../../../target/debug")
+target_link_libraries(visit_expression PUBLIC delta_kernel_ffi)
+target_compile_options(visit_expression PUBLIC)
+
+target_compile_options(visit_expression PRIVATE -Wall -Wextra -Wpedantic -Werror -Wno-strict-prototypes)
+
+# Get info on the OS and platform
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
@zachschuermann  just wanted to have a second look. Slightly yucky, but it's the way I found to target linux for valgrind. Note that macos does not support valgrind.